pkg/corerp/datamodel/environment.go

```diff
@@ -123,3 +128,29 @@ type ProvidersAWS struct {
 	// Scope is the target level for deploying the aws resources
 	Scope string `json:"scope,omitempty"`
 }
+
+// EnvironmentVariables describes structure enabling environment variables to be set.
```

Comment:
Can you please add some more context in this comment.. what are these env variables related to?